config/locales/ja.yml

```
@@ -9,20 +9,23 @@ ja:
         memo: 概要
         author: 著者
         picture: 画像
-  button:
-    edit: 編集
-    back: 戻る
-    show: 詳細
-    destroy: 削除
-    new: 新しい書籍
-    create: 書籍を作る
-  confirmation:
-    本当によろしいですか？
-  title:
-    top: 書籍一覧
-    new: 新しい書籍
-    edit: 書籍の編集
+  book:
+    button:
+      edit: 編集
+      back: 戻る
+      show: 詳細
+      destroy: 削除
+      new: 新規作成
+    title:
+      top: 書籍一覧
+      new: 新しい書籍
+      edit: 書籍の編集
   messages:
     create: 書籍が作成されました。
```

```suggestion
    create: %{model}が作成されました。
```

のように訳文を一部パラメータ化すると、再利用可能な共通の翻訳に昇格させることができます。

---

いくつかの翻訳にパラメータをつけて共通化してみました。